install_linux.sh

```diff
@@ -68,13 +70,13 @@ else
   dest="${TFLINT_INSTALL_PATH:-/usr/local/bin}/"
   echo "Installing /tmp/tflint to ${dest}..."
   
-  if [[ "$(id -u)" == 0 ]]; then SUDO=""; else
+  if [[ "$(id -u)" == 0 ]] || [[ $TFLINT_INSTALL_NO_ROOT == 1 ]]; then SUDO=""; else
```

Comment:
Seems like the script should just test whether it can access the directory as the non-root user and use sudo if not rather than adding an option

Reply:
Good call :+1: https://github.com/terraform-linters/tflint/pull/1184/commits/a80c7639772e978647cb0a34b5eb2d3425b165de